packages/components/src/index.js

```diff
@@ -131,3 +131,6 @@ export {
 } from './higher-order/with-focus-return';
 export { default as withNotices } from './higher-order/with-notices';
 export { default as withSpokenMessages } from './higher-order/with-spoken-messages';
+
+// Utilities
+export { useColorExtract, useControlledState } from './utils/hooks';
```

Comment:
Better address `useControlledState` in a separate PR, otherwise it'll be harder in the future to trace changes back to discussions.